src/components/TopBar/TopBar.vue

```diff
@@ -350,9 +358,9 @@ export default {
 			// Group conversations have mixin in RightSidebar, so should work only for one-to-one
 			handler(newValue) {
 				if (newValue) {
-					this.initialiseGetParticipantsMixin()
+					this.initialiseGetParticipants()
```

Comment:
you can move this watcher from the `TopBar` component, as you have a getter for `isOneToOneConversation` in the composable. Then you don't need to export so many things =)